### DISCUSSION TITLE
posts/P5R

### DISCUSSION THREAD

**Discussion by giscus** [whexy/whexy-blog-comments#23] [General]
# posts/P5R

终于把女神异闻录5皇家版（P5R）通关了。游戏剧本很有意思，但更吸引我的是它独特的配乐风格。

https://www.whexy.com/posts/P5R

**Comment by whexy** [comment]
刚才了解到，心理老师丸喜是 P5R 新加入的角色，后续的剧情都不包含在原版里。原版结局就在打完圣杯之后。所以原版的剧集写的很好！
